config/src/main/java/com/typesafe/config/impl/ConfigNodeRoot.java

```diff
@@ -0,0 +1,38 @@
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigSyntax;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+final class ConfigNodeRoot extends ConfigNodeComplexValue {
+    ConfigNodeRoot(Collection<AbstractConfigNode> children) {
+        super(children);
+    }
+
+    protected ConfigNodeComplexValue value() {
+        for (AbstractConfigNode node : children) {
+            if (node instanceof ConfigNodeComplexValue) {
+                return (ConfigNodeComplexValue)node;
+            }
+        }
+        throw new ConfigException.BugOrBroken("ConfigNodeRoot did not contain a value");
```

Comment:
It does not. 

From what I can tell, if an empty document manages to make its way down to the parser, it will create an empty object and pass that to `ConfigNodeRoot`, so that should always have a value associated with it. I definitely think it's worth writing a test or two to validate this behavior.